appinventor/components/src/com/google/appinventor/components/runtime/File.java

```diff
@@ -319,6 +326,17 @@ public void GotText(String text) {
   }
 
   /**
+   * Event indicating that a request has finished.
+   *
+   * @param text write to the file
+   */
+  @SimpleEvent (description = "Event indicating that the contents of the file have been written.")
+  public void AfterFileSaved(String text) {
```

Comment:
@halatmit Thank you Prof. Hal. Do you mean I should change all the filename ----> fileName inside the  Write(final String filename, final String text, final boolean append) function?